DMInRoblox/DM/Standard/Vector3.dm

```diff
@@ -0,0 +1,4 @@
+/vector3
+	var/x as opendream_compiletimereadonly
+	var/y as opendream_compiletimereadonly
+	var/z as opendream_compiletimereadonly
```

Comment:
Is this `as` on vars an OD-specific thing or have I just been living under a rock?

Reply:
It doesn't do anything in most cases in BYOND, iirc. It's just for `input()` I think.

Reply:
BYOND parses `var/x as ...` without complaint, but it doesn't actually do anything. We take advantage of it to apply compile-time attributes to our built-ins.